apps/web/src/pages/integrations/components/multi-provider/SelectProviderSidebar.tsx

```diff
@@ -1,7 +1,7 @@
 import { useMemo, useState } from 'react';
```

Comment:
could you please export the image `empty-provider-search.png` as the SVG, so it will weight less and scale better?